### DISCUSSION TITLE
Playlists - Option to set maximum attempts "per beatmap". + More noticeable UI implementation.

### DISCUSSION THREAD

**Discussion by HyperSlayer72** [ppy/osu#22685] [Ideas]
**1.** Heyo. At the moment it seems that the maximum attempts option gives the player an amount of attempts to use across all beatmaps. This can remain an option, however we also need alternatives. A good start would be an option to set maximum attempts "per beatmap". Meaning that a setting of 3, would grant 3 attempts per beatmap, once a player runs out of attempts on a beatmap the best submission of those 3 would be used, and the player has to move onto the next beatmap in the playlist.

I also feel this likely should be the default, as my expectations were definitely subverted when I made my first playlist. I saw the maximum attempts option and automatically thought, ah yes, how many tries do I want for each song before no more are allowed. Only to then make a playlist with 10 songs, and a maximum of 7 attempts :/ welp... time to wait a week for it to expire. As has been discussed elsewhere we definitely need to be able to edit a playlist after creating it (at least before anyone else joins in).

I don't feel the current implementation needs to go away, in fact I think being able to limit attempts below the total available beatmaps grants players to chance to carefully consider which maps they are most adept at before attempting a score. 

If you want to fledge the setting out more, there's lots of different permutations available.

2. You could make the competition more cutthroat by utilizing the last submitted attempt, meaning that a player has to carefully consider trying a beatmap again, weighting the risk of potentially setting a worse score. 

3. You could also take a weighted average of all attempts, meaning that a player who successfully sets a great play on their first try would be rewarded for their greater consistency and sight read ability. Over a player who needed 10 tries to set the same score.

     **2**. I do feel the maximum attempts remaining needs to be more noticeable, as I didn't even notice them counting down until I ran out of attempts.  The attempts remaining likely needs to integrate with the beatmap list somehow.

![image](https://user-images.githubusercontent.com/20193169/219879681-b54c76de-5c31-471c-96fb-1549b7eaadbe.png)

**Comment by WalmartWhiteCat** [comment]
I've actually been thinking about this a lot. I think it would be cool.
